src/ontology/pato-edit.obo

```diff
@@ -22035,6 +22035,56 @@ xref: NLX:378
 is_a: PATO:0010006 ! cell morphology
 created_by: http://orcid.org/0000-0001-7258-9596
 
+[Term]
+id: PATO:0070028
+name: extratelencephalic projection
+def: "A neuron projection quality that is a characteristic of neurons with both soma within the structures derived from the telencephalon and projections to structures not derived from the telencephalon." [DOI:10.1101/2020.10.19.343129]
+xref: ILX:0770101
+is_a: PATO:0070033 ! neuron projection quality
+created_by: http://orcid.org/0000-0001-7258-9596
+
+[Term]
+id: PATO:0070029
+name: corticothalamic projection
+def: "A neuron projection quality that is a characteristic of neurons with soma in the cerebral cortex and projections to the thalamus." [DOI:10.1101/2020.10.19.343129]
+synonym: "CT projection" EXACT []
+xref: ILX:0770102
+is_a: PATO:0070033 ! neuron projection quality
+created_by: http://orcid.org/0000-0001-7258-9596
+
+[Term]
+id: PATO:0070030
+name: near projection
+def: "A neuron projection quality that is a characteristic of neurons with projections that terminate locally rather than distantly." [DOI:10.1101/2020.10.19.343129]
+synonym: "NP" EXACT []
+xref: ILX:0770103
+is_a: PATO:0070033 ! neuron projection quality
+created_by: http://orcid.org/0000-0001-7258-9596
+
+[Term]
+id: PATO:0070031
+name: medulla projection
+def: "A neuron projection quality that is a characteristic of neurons with projections that terminate in the medulla." [DOI:10.1101/2020.10.19.343129]
```

Comment:
medulla ->  medulla oblongata ?

There are lots of medullas in anatomy, I think above is the correct one.

I also think that a tighter definition would be better "neurons that project to the medulla oblongata from other brain regions".   (For our purposes coritical medulla projecting might be even better - but we would need to change the term name).

GCI: bearer_of some 'medulla projecting' subClassOf  'axon synapses in' some 'medulla oblongata'

Reply:
yes, its the medulla oblongata, my bad, neuroscientists assumptions/bad habits to equate medulla to medulla oblongata i guess :)